src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -96,13 +96,12 @@ class SettingsMigration : Activity {
         .filter { it.llm == null }
         .forEach {
           val (model, provider, title) =
-              modelToProviderAndTitle.getOrDefault(it.model, Triple("", "Default", "Default"))
+              modelToProviderAndTitle.getOrDefault(it.model, Triple(it.model, null, null))
```

Comment:
LGTM, but maybe let's do the same change in `createFromState`

Reply:
👍